rts/idris_rts.c

```diff
@@ -417,11 +409,10 @@ void dumpStack(VM* vm) {
 void dumpVal(VAL v) {
     if (v==NULL) return;
     int i;
-    if (ISINT(v)) {
-        printf("%d ", (int)(GETINT(v)));
-        return;
-    }
     switch(GETTY(v)) {
+    case CT_INT:
+        printf("%d ", (int)(GETINT(v)));
```

Comment:
This was wrong before, but this always prints an int as a 32-bit number, might as well fix that.